<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LF9C4GT" failures="2" tests="2" name="LoginAPITest" time="4.123" errors="0" timestamp="2025-07-04T16:19:42 IST">
  <testcase classname="tests.LoginAPITest" name="testNegativeLogin" time="3.448">
    <failure type="java.lang.AssertionError" message="expected [400] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [401]
at tests.LoginAPITest.testNegativeLogin(LoginAPITest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testNegativeLogin -->
  <testcase classname="tests.LoginAPITest" name="testPositiveLogin" time="0.595">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at tests.LoginAPITest.testPositiveLogin(LoginAPITest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testPositiveLogin -->
</testsuite> <!-- LoginAPITest -->
